jobmon_client/src/jobmon/client/task_group.py

```diff
@@ -213,7 +213,7 @@ def interleave_downstream(
         subsetter: Optional[Dict] = None,
         upstream_subsetter: Optional[Dict] = None,
     ):
-        pass
+        raise NotImplementedError
```

Comment:
i realized I forgot to implement or write tests for this; not sure it's in scope. I can though if you'd like to me too. I'm not sure how thorough we want the test suite for it to be

Reply:
You can always write a ticket to implement this last method and write a test! What you've done is already quite comprehensive

Reply:
Also, this method could be almost trivial if you just do something like `downstream_group.interleave_upstream(self)` right?

Reply:
yeah I think implementation is going to pretty easy! Tests are going to be more work, depending on how complete we want them to be